libdrgn/python/module.c

```diff
@@ -155,10 +201,11 @@ DRGNPY_PUBLIC PyMODINIT_FUNC PyInit__drgn(void)
 	if (add_module_constants(m) == -1)
 		goto err;
 
-	FaultError = PyErr_NewExceptionWithDoc("_drgn.FaultError",
-					       drgn_FaultError_DOC, NULL, NULL);
-	if (!FaultError)
+	PyExc_FaultError.tp_base = (PyTypeObject*)PyExc_Exception;
+	if (PyType_Ready(&PyExc_FaultError) < 0)
 		goto err;
+	Py_INCREF(&PyExc_FaultError);
+	FaultError = (PyObject *)&PyExc_FaultError;
```

Comment:
We don't need the `FaultError` variable anymore, let's get rid of it and use `&FaultError_type` casted to the appropriate types in the couple of places we currently reference `FaultError`